### PR TITLE
Xmlschema

### PR DESCRIPTION
**Pull request by hakuturu583**
Fix problems described in this PR.
Sorry for missing ping.
https://github.com/ros/rosdistro/pull/26913

I want to add xmlschema to rosdistro.

Pypy package: https://pypi.org/project/xmlschema/

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -5281,16 +5281,16 @@ python-xlrd:
   debian: [python-xlrd]
   fedora: [python-xlrd]
   ubuntu: [python-xlrd]
-python-xmlplain-pip:
+python-xmlschema:
```
Comment (tfoote) [inline rosdep/python.yaml:5284]: you seem to be removing an existing key here and overwriting it.

File: rosdep/python.yaml [hunk 2]
```diff
@@ -6874,6 +6874,16 @@ python3-wxgtk4.0:
   fedora: [python3-wxpython4]
   gentoo: [dev-python/wxpython]
   ubuntu: [python3-wxgtk4.0]
+python3-xmlschema-pip:
+  debian:
+    pip:
+      packages: [xmlplain]
```
Comment (tfoote) [inline rosdep/python.yaml:6880]: This looks like the new set of rules that you're adding, but they appear to be referencing  a different packge.

**Review by tfoote — CHANGES_REQUESTED** [review]
This PR needs a little bit of cleanup to just be inserting the new rule mentioned in the description.

File: rosdep/python.yaml (hunk 1 above)
Comment (tfoote) [inline rosdep/python.yaml:5284]: You're still removing the python-xmlplain-pip. Please maintain that key and add the new keys.

**Comment by hakuturu583** [comment]
I add python-xmlplain-pip package

File: rosdep/python.yaml [hunk 3]
```diff
@@ -5291,6 +5291,16 @@ python-xmlplain-pip:
   ubuntu:
     pip:
       packages: [xmlplain]
+python-xmlschema:
```
Comment (clalancette) [inline rosdep/python.yaml:5294]: Since this is a pip-only package, it should be: `python3-xmlschema-pip`

**Comment by hakuturu583** [comment]
Sorry, I took a lot of mistakes in this branch, so I re-create PR about xmlschema package.

**Pull request closed without merging**